From 54a9bd28a307dd74df044d779ca85adcf36aa202 Mon Sep 17 00:00:00 2001
From: Joe MacDonald <joe.macdonald@windriver.com>
Date: Tue, 18 Jun 2013 10:05:21 -0400
Subject: [PATCH] Fix the dependency issue

Upstream-Status: Pending

autofs's lib sources have a dependency on a number of files that are
generated by rpcgen during buildtime

Signed-off-by: Roy.Li <rongqing.li@windriver.com>
Signed-off-by: Joe MacDonald <joe.macdonald@windriver.com>
---
 lib/Makefile |    1 +
 1 file changed, 1 insertion(+)

diff --git a/lib/Makefile b/lib/Makefile
index 5418009..bff4e1d 100644
--- a/lib/Makefile
+++ b/lib/Makefile
@@ -75,3 +75,4 @@ install: all
 clean:
 	rm -f $(LIB) $(RPCS) $(OBJS) $(YACCSRC) *.output *~
 
+$(OBJS): $(RPCS)
-- 
1.7.10.4

